respa_admin/tests/test_resource_forms.py

```diff
@@ -42,12 +39,8 @@ def test_period_formset_with_invalid_days_data(valid_resource_form_data):
     request = RequestFactory().post(NEW_RESOURCE_URL, data=data)
     period_formset_with_days = get_period_formset(request)
     assert period_formset_with_days.is_valid() is False
-    assert period_formset_with_days.errors == [
-        {'__all__': ['Tarkista aukioloajat.']},
-    ]
-    assert period_formset_with_days.forms[0].days.errors == [
-        {'weekday': ['Tämä kenttä vaaditaan.']},
-    ]
+    assert period_formset_with_days.errors != []
+    assert period_formset_with_days.forms[0].days.errors != []
```

Comment:
Same here. You should still check that the dict keys are exactly `__all__` and `weekday` in those cases.